### DISCUSSION TITLE
特定資源個例討論

### DISCUSSION THREAD

**Discussion by NeKoOuO** [open-ani/animeko#186] [Q&A] [answered]
延續 #133 的討論

感覺這是個長久的問題

**Comment by NeKoOuO** [comment]


**Reply by Him188** [reply]
推のし子实际上是 NC-RAWS, 他们每个季度都会换一个名字

**Reply by NeKoOuO** [reply]
挖哩 那無解了阿 還是專門設這個識別詞

**Reply by Him188** [reply]
理论上我的规则是可以适配它这个格式的, 之后可以看看

**Comment by Him188** [comment]
https://www.dmhy.org/topics/list?keyword=%E6%A8%B1trick

空格问题
```
[[愛戀&漫猫字幕社]櫻Trick Sakura Trick 01-12 avc_ flac mkv繁體內嵌合集（急招時軸）](https://www.dmhy.org/topics/view/656570_Trick_Sakura_Trick_01-12_avc_flac_mkv.html)

[[DMG&Hakugetsu&VCB-Studio] Sakura Trick / 樱Trick 10-bit 1080p BDRip [Reseed Fin]](https://www.dmhy.org/topics/view/451596_DMG_Hakugetsu_VCB-Studio_Sakura_Trick_Trick_10-bit_1080p_BDRip_Reseed_Fin.html) 
```

缺失集数
```
[【华盟字幕社】[Sakura_Trick][樱Trick][BDRip][简繁日外挂][庆祝华盟字幕社成立十周年]](https://www.dmhy.org/topics/view/367159_Sakura_Trick_Trick_BDRip.html)
```

**Comment by NeKoOuO** [comment]
特典映像
![Screenshot_2024-05-26-10-38-48-817_me.him188.ani.jpg](https://github.com/open-ani/ani/assets/111511925/beba5c3d-4833-43f4-87ef-4866dd938382)

**Comment by NeKoOuO** [comment, marked as answer]
史萊姆匹配錯誤 (12)
![Screenshot_2024-06-22-22-52-20-198_me.him188.ani.jpg](https://github.com/open-ani/ani/assets/111511925/632df1fc-7697-49b1-b81c-9d5a99696128)

**Reply by Him188** [reply]
这是因为弹弹 play 里面还没有这一集我估计

**Reply by NeKoOuO** [reply]
> 这是因为弹弹 play 里面还没有这一集我估计

難怪前兩集匹配的到
